examples/connext_dds/instance_statistics/README.md

```diff
@@ -0,0 +1,52 @@
+# Example Code: Instance Statistics
+
+Example about getting instance statistics.
```

Comment:
Example showing how to obtain instance statistics.

(I cannot use the github suggestion feature on this review since it is targetting master)